com.visualstudio.code.metainfo.xml

```diff
@@ -8,8 +8,9 @@
   <url type="homepage">https://code.visualstudio.com</url>
   <summary>Visual Studio Code. Code editing. Redefined.</summary>
   <description>
-    <p>Visual Studio Code is a new choice of tool that combines the simplicity of a code editor with what developers need for the core edit-build-debug cycle.</p>
+    <p>IMPORTANT: To use certain features (like the integrated terminal) in this flatpacked version, you will need to <a href="https://github.com/flathub/com.visualstudio.code#visual-studio-code-flatpak-" target="_blank">follow the quick README here</a>.</p>
```

Comment:
I'd rather move it below the usual description. It should remain visible and just seems better organizationally.

Reply:
Got it! Was worried it'd get lost below the fold on Gnome Software: 
![image](https://user-images.githubusercontent.com/20881877/177635284-911a68ae-0e57-4d48-8c42-1a5e87146b79.png)

Can I move it to second place, before we disclaim that the build is unofficial?

Reply:
Yeah, that sounds good to me.